diffkemp/simpll/DifferentialFunctionComparator.h

```diff
@@ -138,11 +138,14 @@ class DifferentialFunctionComparator : public FunctionComparator {
     mutable std::vector<std::pair<const PHINode *, const PHINode *>>
             phisToCompare;
     mutable std::unordered_map<const Value *, const Value *>
-            ignoredInstructions;
+            replacedInstructions;
+    mutable std::set<const Value *> ignoredInstructions;
```

Comment:
```suggestion
    mutable std::unordered_set<const Value *> ignoredInstructions;
```
to unify with `replacedInstructions`

Reply:
Fixed.